scipy/stats/_qmc.py

```diff
@@ -1739,8 +1724,25 @@ def reset(self) -> MultivariateNormalQMC:
             Engine reset to its base state.
 
         """
-        super().reset()
         self.engine.reset()
+        self.num_generated = 0
+        return self
+
+    def fast_forward(self, n: IntNumber) -> QMCEngine:
+        """Fast-forward the sequence by `n` positions.
+
+        Parameters
+        ----------
+        n : int
+            Number of points to skip in the sequence.
+
+        Returns
+        -------
+        engine : MultivariateNormalQMC
+            The MultivariateNormalQMC engine itself.
+
+        """
+        self.random(n=n)
```

Comment:
I think it should be
```suggestion
        self.engine.fast_forward(n=n)
```

Reply:
Sure.